assets/charts/control-plane/kubernetes/templates/kube-scheduler-role-binding.yaml

```diff
@@ -10,3 +10,16 @@ subjects:
 - kind: ServiceAccount
   name: kube-scheduler
   namespace: kube-system
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: system:configmaps-reader
```

Comment:
Name should probably be unique. Like `kube-scheduler-extension-apiserver-authentication-reader` or something.